Doc/source/designspaceLib/index.rst

```diff
@@ -120,6 +120,34 @@ any of the UFOs. If the lib key is empty or not present in the Designspace, all
 glyphs should be exported, regardless of what the same lib key in any of the
 UFOs says.
 
+public.fontInfo
+-----------------------
+
+This lib key, when included in the ``<lib>`` element inside an ``<instance>`` or 
+``<variable-font>`` tag, allows for direct manipulation of font info data in 
+instances and variable fonts. The lib value must follow the 
```

Comment:
Should we allow this lib key to also be present in a third place: the root `<lib>` element of the whole designspace?

That would support two use-cases:
* it would work for DS version 4, in that case it would specify the fontinfo of the VF that covers the whole designspace (as in DS v4, one DS == one VF)
* it would one more level of priority when looking up a value:
  1. use the value from `<variable-font>` or `<instance>`'s lib public.fontInfo
  2. use the value from the `<varaiable-font>` or `<instance>` XML attributes for names
  3. (NEW) use the value from the designspace's root `<lib>`'s public.fontInfo
  4. use the value from the source UFO's fontinfo.plist

Reply:
> Should we allow this lib key to also be present in a third place: the root <lib> element of the whole designspace?

yeah +1 on that